docs/release-notes/NuGet-6.2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: NuGet 6.2 Release Notes
+description: Release notes for NuGet 6.2 including new features, bug fixes, and DCRs.
+author: martinrrm
+ms.author: mruizmares
+ms.date: 5/9/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.2 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.2.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.2](https://visualstudio.microsoft.com/downloads/) | [6.0.202](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.2
+
+* Add TFM for .NET nanoFramework - [#10800](https://github.com/NuGet/Home/issues/10800)
+
+* [Feature]: Require package source mapping when using CPM - [#11505](https://github.com/NuGet/Home/issues/11505)
+
+* [Feature]: Allow overriding a centrally defined package version - [#11516](https://github.com/NuGet/Home/issues/11516)
+
+* [Feature]: Add support for a dedicated environment variable providing the NuGetScratch path. - [#11671](https://github.com/NuGet/Home/issues/11671)
+
+* [Feature]: Add IVsNuGetProjectUpdateEvents in Visual Studio, reporting of restore changes for PackageReference based projects.  - [#9782](https://github.com/NuGet/Home/issues/9782)  - [See documentation](https://docs.microsoft.com/en-us/nuget/visual-studio-extensibility/nuget-api-in-visual-studio#ivsnugetprojectupdateevents-interface)
```

Comment:
Is `See documentation` use correct?

Reply:
That's one way of doing it. It's ok imo.

Reply:
you need to use a relative link to the api, not an http link.